---
permalink: /
title: "Welcome to [Lin Liu (刘林)](https://linliu-stats.github.io/)'s homepage"
excerpt: "Personal Information"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am an Assistant Professor at the [Institute of Natural Sciences (INS)](https://ins.sjtu.edu.cn/) at [Shanghai Jiao Tong University (SJTU)](http://en.sjtu.edu.cn/). I am also affiliated with the [School of Mathematical Sciences](http://math.sjtu.edu.cn/) and the [SJTU-YALE Joint Center for Biostatistics and Data Science](http://transmed.sjtu.edu.cn/ylhome.do?method=getHomeList&version=en).
 
I graduated from the [Department of Biostatistics](https://www.hsph.harvard.edu/biostatistics/) at [Harvard University](https://www.harvard.edu/) in 2018. My advisors are [Professor Franziska Michor](http://michorlab.dfci.harvard.edu/) and [Professor James M. Robins](https://www.hsph.harvard.edu/james-robins/). My current research lies in nonparametric, semiparametric and high-dimensional statistics, robust statistical methods, causal inference, computational and mathematical biology.

I am also interested in the theory of deep learning, estimation and inference in inverse problems and applying causal inference tools in biomedical research.

I obtained my undergraduate degree from the [School of Life Sciences at Tongji University](https://life.tongji.edu.cn/lifeen/), under the supervision of [Professor Yong Zhang](https://zhanglab.tongji.edu.cn).

You can reach me by email: [linliu@alumni.tongji.edu.cn](linliu@alumni.tongji.edu.cn) or [linliu@sjtu.edu.cn](linliu@sjtu.edu.cn)

Selected Papers
======
(You can find all my papers on my [Google Scholar profile](https://scholar.google.com/citations?user=2xESgioAAAAJ&hl=en).) (_Italic_: co-first authorship; #: (co)-corresponding authors)

**_Working papers_**:

LL, and Chang Li. [New semiparametric efficient, numerically stable higher-order influence function estimators.]() Under preparation.

Siqi Xu, LL, and Zhonghua Liu. [DeepMed: Semiparametric causal mediation analysis with debiased deep learning.](http://linliu-stats.github.io/files/DeepMed.pdf) Submitted.

James M. Robins, Lingling Li, LL, Rajarshi Mukherjee, Eric Tchetgen Tchetgen, and Aad van der Vaart. [Corrigenda to "Minimax estimation of a functional on a structured high-dimensional model" by James M. Robins, Lingling Li, Rajarshi Mukherjee, Eric Tchetgen Tchetgen, and Aad van der Vaart.]() Available upon request.

LL, Rajarshi Mukherjee, Whitney Newey, and James M. Robins. [Semiparametric efficient empirical higher-order influence function estimators.](http://linliu-stats.github.io/files/Revision_Arxiv_Empirical_HOIF.pdf) Under review.

LL, Rajarshi Mukherjee, and James M Robins. [Can we tell if the justification of the validity of Wald confidence intervals of doubly robust functionals may be incorrect, without assumptions?](http://linliu-stats.github.io/files/Oct24-Assumption-free%20test%20econ%20(ref%20together).pdf) Under revision.

<!--- Siqi Xu, LL#, Zhonghua Liu#. DeepMed: Semiparametric causal mediation analysis with deep learning. Under preparation.--->

**_Statistical theory_**:

LL, Rajarshi Mukherjee, James M Robins, and Eric Tchetgen Tchetgen. [Adaptive estimation of nonparametric functionals.](http://linliu-stats.github.io/files/Adaptive_Arxiv_Update.pdf) (2021). Journal of Machine Learning Research 22 (99), 1-66.

LL, Rajarshi Mukherjee, and James M Robins. [On nearly assumption-free tests of nominal confidence interval coverage for causal parameters estimated by machine learning.](https://projecteuclid.org/journals/statistical-science/volume-35/issue-3/On-Nearly-Assumption-Free-Tests-of-Nominal-Confidence-Interval-Coverage/10.1214/20-STS786.short) (2020). Statistical Science 35 (3), 518-539. ([arXiv: 1904.04276](https://arxiv.org/abs/1904.04276))

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; See the [Discussion](https://projecteuclid.org/journals/statistical-science/volume-35/issue-3/Discussion-of-On-Nearly-Assumption-Free-Tests-of-Nominal-Confidence/10.1214/20-STS796.short) ([arXiv: 2006.09613](https://arxiv.org/abs/2006.09613)) of our paper by [Edward H. Kennedy](http://www.ehkennedy.com/), [Siva Balakrishnan](http://www.stat.cmu.edu/~siva/), and [Larry Wasserman](https://www.stat.cmu.edu/~larry/) and our [Rejoinder](https://projecteuclid.org/journals/statistical-science/volume-35/issue-3/Rejoinder--On-nearly-assumption-free-tests-of-nominal-confidence/10.1214/20-STS804.short) ([arXiv: 2008.03288](https://arxiv.org/abs/2008.03288)).

**_Statistical methodology and causal inference_**:

LL, Zach Shahn, James M Robins and Andrea Rotnitzky. [Efficient estimation of optimal regimes under a no direct effect assumption.](https://www.tandfonline.com/doi/full/10.1080/01621459.2020.1856117) (2021). Journal of the American Statistical Association 116 (533), 224-239.

**_Statistical Computing_**:

Lei Li, LL, Yuzhou Peng. [A splitting Hamiltonian Monte Carlo method for efficient sampling.](https://arxiv.org/abs/2105.14406) (2021+). Submitted.

**_Mathematical biology_**:

Kimiyo N Yamamoto, LLL, Akira Nakamura, Hiroshi Haeno, and Franziska Michor. [Stochastic evolution of pancreatic cancer metastases during logistic clonal expansion.](https://ascopubs.org/doi/abs/10.1200/CCI.18.00079) (2019). JCO Clinical Cancer Informatics 3: 1-11. 

Helena A Yu, Camelia Sima, Daniel Feldman, LLL, Bhavapriya Vaitheesvaran, Justin Cross, Charles M Rudin, Mark G Kris, William Pao, Franziska Michor, and Gregory J Riely. [Phase 1 study of twice weekly pulse dose and daily low-dose erlotinib as initial treatment for patients with EGFR-mutant lung cancers.](https://academic.oup.com/annonc/article-abstract/28/2/278/2676878) (2019). Annals of Oncology 28 (2): 278-284.

LLL, Justin Brumbaugh, Ori Bar-Nur, Zachary Smith, Matthias Stadtfeld, Alexander Meissner, Konrad Hochedlinger, and Franziska Michor. [Probabilistic modeling of reprogramming to induced pluripotent stem cells.](https://www.sciencedirect.com/science/article/pii/S2211124716316540) (2016). Cell Reports 17 (12): 3395-3406.

_Philipp M Altrock_, _LLL_, and Franziska Michor. [The mathematics of cancer: integrating quantitative models.](https://www.nature.com/articles/nrc4029) (2015). Nature Reviews Cancer 15 (12): 730-745.

_Jasmine Foo_, _LLL_, Kevin Leder, Markus Riester, Yoh Iwasa, Christoph Lengauer, and Franziska Michor. [An evolutionary approach for identifying driver mutations in colorectal cancer.](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004350) (2015). PLoS Computational Biology 11 (9): e1004350.

**_Statistical and Machine Learning Applications_**:

Sheng'en S. Hu, LL, Qi Li, Wenjing Ma, Michael J. Guertin, Clifford A. Meyer, Ke Deng, Tingting Zhang, Chongzhi Zang (2021+). Accurate estimation of intrinsic biases for improved analysis of chromatin accessibility sequencing data using SELMA. Under review. [bioRxiv link](https://www.biorxiv.org/content/10.1101/2021.10.22.465530v1.full.pdf).

Jeremy R. Glissen Brown, Nabil M. Mansour, Pu Wang, Maria Aguilera Chuchuca, Scott B. Minchenberg, Madhuri Chandnani, LL, Seth A. Gross, Neil Sengupta, Tyler M. Berzin. [Deep learning computer-aided polyp detection reduces Adenoma Miss Rate:  A U.S. multi-center randomized tandem colonoscopy study (CADeT-CS Trial).](https://www.cghjournal.org/article/S1542-3565(21)00973-3/pdf) (2021+). In press in Clinical Gastroenterology and Hepatology.

Michael J Leone, Haoqi Sun, Christine L Boutros, LL, Elissa Ye, Lee Sullivan, Robert J Thomas, Gregory K Robbins, Shibani S Mukerji, M Brandon Westover. [HIV Increases Sleep-based Brain Age Despite Antiretroviral Therapy.](https://academic.oup.com/sleep/advance-article-abstract/doi/10.1093/sleep/zsab058/6204183) (2021). Sleep 44 (8): zsab058.

Sheng'en Hu, Dawei Huo, Zhaowei Yu, Yujie Chen, Jing Liu, LL, Xudong Wu, and Yong Zhang. [ncHMR detector: a computational framework to systematically reveal non-classical functions of histone modification regulators.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01953-0) (2020). Genome Biology 21 (1): 48.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ncHMR detector: [software link](https://github.com/TongjiZhanglab/ncHMR_detector/); [webserver link](http://compbio-zhanglab.org/ncHMR_detector/index.php)

_Kyle S Smith_, _LLL_, Shridar Ganesan, Franziska Michor, and Subhajyoti De. [Nuclear topology modulates the mutational landscapes of cancer genomes.](https://www.nature.com/nsmb/journal/v24/n11/abs/nsmb.3474.html) (2017). Nature Structural & Molecular Biology 24 (11): 1000-1006.

Michalina Janiszewska, LL, Vanessa Almendro, Yanan Kuang, Cloud Paweletz, Rita A Sakr, Britta Weigelt, Ariella B Hanker, Sarat Chandarlapaty, Tari A King, Jorge S Reis-Filho, Carlos L Arteaga, So Yeon Park, Franziska Michor, and Kornelia Polyak. [In situ single-cell analysis identifies heterogeneity for PIK3CA mutation and HER2 amplification in HER2-positive breast cancer.](https://www.nature.com/articles/ng.3391/) (2015). Nature Genetics 47 (10): 1212-1219.

LL, Subhajyoti De, and Franziska Michor. [DNA replication timing and higher-order nuclear organization determine single-nucleotide substitution patterns in cancer genomes.](https://www.nature.com/articles/ncomms2502) (2013). Nature Communications 4: 1502.

_LL_, _Yiqian Zhang_, Jianxing Feng, Ning Zheng, Junfeng Yin, and Yong Zhang. [GeSICA: Genome segmentation from intra-chromosomal associations.](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-164) (2012). BMC Genomics 13 (1): 164.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; GeSICA: [software link](https://zhanglab.tongji.edu.cn/softwares/GeSICA/index.html)

Qi Liu, Han Zhou, LL, Xi Chen, Ruixin Zhu, and Zhiwei Cao. [Multi-target QSAR modelling in the analysis and design of HIV-HCV co-inhibitors: an in-silico study.](https://link.springer.com/article/10.1186/1471-2105-12-294) (2011). BMC Bioinformatics 12 (1): 294.

**_Miscellaneous_**:

LL. [Book Review: _Matrix-Based Introduction to Multivariate Data Analysis, 2nd Edition_ by Kohei Adachi.](https://onlinelibrary.wiley.com/doi/full/10.1111/biom.13566) (2021). Biometrics 77 (4): 1498-1500.
